otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -24,6 +24,11 @@ data:
     networkobservabilityHubble = true
     networkobservabilityCilium = true
     prometheuscollectorhealth = false
+    controlplane-apiserver = true
+    controlplane-cluster-autoscaler = false
+    controlplane-kube-scheduler = false
+    controlplane-kube-controller-manager = false
+    controlplane-etcd = true
```

Comment:
Thanks Kaveesh, will this onboard customers to public preview if they apply this configmap? Or is there still another setting they need to enable? Or for public preview is this going to be enabled for all customers?

Reply:
Here is a link to their onboarding documentation: https://learn.microsoft.com/en-us/azure/aks/monitor-control-plane-metrics


Applying this configmap will not onboard the said customers as they'll need to have enabled the feature flag.

If a customer has enabled the required feature flag on their subscription, then onboarding to Managed Prometheus will also onboard them in the underlay clusters.

Reply:
Sounds good, thanks!